### PR TITLE
Add definition and perspective for 'KPI'

### PR DESCRIPTION
**Pull request by Amaka202**
## This PR fixes...

The lack of definition and perspective for 'KPI'

## What I did...

- Add dictionary definition for 'KPI'
- Add Software developer perspective for 'KPI'

## How to test...

1. Navigate to /dictionary
1. Search 'KPI'
1. See if the definition and perspective show up under  the term 'KPI'
2. Check grammar and typos

## I learned...

That KPI is more of a business term other than a software term

### CODE REVIEW

File: src/assets/content/dictionary/kpi.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ---
 title: KPI
-definition: ''
-sources: []
-perspectives: []
+definition: Key Performance Indicators (KPIs) is a critical (key) indicator of progress toward an intended result. A KPI provides a focus for strategic and operational improvement, create an analytical basis for decision making and help focus attention on what matters most.
+sources:
+- sourceurl: https://kpi.org/KPI-Basics
+perspectives:
+- meaning: a metric or indicator to measure the work done by software teams in order to remain accountable to set objectives. Eg lines of code, bugs, task completion dealines
```
Comment (alliequintano) [inline src/assets/content/dictionary/kpi.md]:
```suggestion
- meaning: a way to measure the work I have done in order to remain accountable to set objectives
```

**Review by alliequintano — APPROVED** [review]
Hi @Amaka202! I've made a small suggestion to improve the meaning you've provided. I'll be happy to approve with this change. Thank you!

**Review by alliequintano — CHANGES_REQUESTED** [review]
Hi @Amaka202! I've made a small suggestion to improve the meaning you've provided. I'll be happy to approve with this change. Thank you!

**Comment by Amaka202** [comment]
> Hi @Amaka202! I've made a small suggestion to improve the meaning you've provided. I'll be happy to approve with this change. Thank you!

Thank you so much i have made the updates @alliequintano

**Review by novellac — APPROVED** [review]

**Pull request merged**